include/boost/json/impl/pointer.ipp

```diff
@@ -0,0 +1,329 @@
+//
+// Copyright (c) 2021 Dmitry Arkhipov (grisumbras@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/json
+//
+
+#ifndef BOOST_JSON_IMPL_POINTER_IPP
+#define BOOST_JSON_IMPL_POINTER_IPP
+
+#include <boost/json/pointer.hpp>
+
+BOOST_JSON_NS_BEGIN
+
+namespace detail {
+
+
+class pointer_token
+{
+public:
+    class iterator;
+
+    pointer_token(
+        string_view::const_iterator b,
```

Comment:
kind of a long winded way to write `char const*` no?

Reply:
Apparently `string_view::const_iterator` is not `char const*` on msvs 14. I guess, I could try changing it from the other side and use `string_view::const_iterator` less.

Reply:
What is the type on msvs 14.1?